docs/LV_client/tutorials/Logger.md

```diff
@@ -38,11 +43,17 @@ This VI will produce a text file for:
 <img class="lg_img" src="../logger.png"/>
```

Comment:
Would it be possible to get a second picture of the wire details for visual learners? The inputs and outputs are well named, but my instinct was to connect them to how the picture looks first.

Reply:
I updated this picture, let me know if this is better.

Reply:
As you mention, connecting the annotation only matters if you have annotation set to true but I do see how it can be confusing.